src/components/VContentSwitcher/VSearchTypeRadio.vue

```diff
@@ -13,17 +13,19 @@
 
 <script>
 import { computed, defineComponent } from '@nuxtjs/composition-api'
+
 import {
   ALL_MEDIA,
   AUDIO,
   IMAGE,
   supportedSearchTypes,
 } from '~/constants/media'
+
 import VIcon from '~/components/VIcon/VIcon.vue'
 
-import audioIcon from 'assets/icons/audio-wave.svg'
-import imageIcon from 'assets/icons/image.svg'
-import allIcon from 'assets/icons/all-content.svg'
+import audioIcon from '~/assets/icons/audio-wave.svg'
```

Comment:
How did this get in 🤦 . I think it raises an error in tests when there is no leading `~/`

Reply:
Interesting! Well our linter will prevent it from now :slightly_smiling_face: